src/test/java/org/superbiz/ColorResourceTest.java

```diff
@@ -42,10 +47,11 @@
  * A second value-add is it is possible to build WebArchives that are slim and trim and therefore
  * isolate the functionality being tested.  This also makes it easier to swap out one implementation
  * of a class for another allowing for easy mocking.
- *
  */
 @RunWith(Arquillian.class)
-public class ColorServiceTest extends Assert {
+@RunAsClient
```

Comment:
That's exactly what `@RunAsClient` does. If it's set on the class, it automatically implies testable=false on all deployments. You only need the testable flag if you're trying to mix client and server test cases in the same test class.